<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="dateTimeLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Add machine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);
border-radius: 10px;
}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="machineModelField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="machineOperatorField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Operator ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dialogAddMachineButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);
border-radius: 10px;
}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Planned maintanence</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarMachine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
